.idea/codeStyleSettings.xml

```diff
@@ -3,10 +3,28 @@
   <component name="ProjectCodeStyleSettingsManager">
     <option name="PER_PROJECT_SETTINGS">
       <value>
-        <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
-        <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
+        <option name="OTHER_INDENT_OPTIONS">
```

Comment:
https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml

On Mon, Apr 25, 2016 at 4:02 PM, meltsufin <notifications@github.com> wrote:

> In .idea/codeStyleSettings.xml
> <https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/pull/17#discussion_r60978319>
> :
>
> > @@ -3,10 +3,28 @@
> >    <component name="ProjectCodeStyleSettingsManager">
> >      <option name="PER_PROJECT_SETTINGS">
> >        <value>
> > -        <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
> > -        <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
> > +        <option name="OTHER_INDENT_OPTIONS">
>
> Where is the official one? Some things like the new line before package
> wasn't working.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/pull/17/files/96806a1bdd2effb12ffacce6aeda254d24d2c011#r60978319>
>